WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -38,20 +40,31 @@ struct ReviewsDashboardCard: View {
                 .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
                 .shimmering(active: viewModel.syncingData)
             Divider()
+                .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                .shimmering(active: viewModel.syncingData)
 
-            ForEach(Array(dummyData.enumerated()), id: \.element.reviewID) { index, review in
-                ReviewRow(for: review, isLastItem: index == dummyData.count-1)
-            }
-            Divider()
-            viewAllReviewsButton
-                .padding(.horizontal, Layout.padding)
+            if viewModel.data.isNotEmpty {
+                ForEach(Array(viewModel.data.enumerated()), id: \.element.review.reviewID) { index, reviewViewModel in
+                    ReviewRow(for: reviewViewModel, isLastItem: index == dummyData.count-1)
```

Comment:
```suggestion
                    ReviewRow(for: reviewViewModel, isLastItem: index == viewModel.data.count - 1)
```

Reply:
Thanks! I fixed this in https://github.com/woocommerce/woocommerce-ios/pull/12752/commits/34094ecff5fd92c7ef1e7563a18c9ca874385c48 as I noticed it also myself before reading your comment.